outlines/welcome.md

```diff
@@ -0,0 +1,29 @@
+1. Welcome to the Meteor guide
```

Comment:
I'd add something to the top that says "this isn't the best place to get started with Meteor if you're just discovering it. Go do the intro tutorial"